### ISSUE TITLE
Wiki Spoiler-tier part group page isnt actually considered Spoiler-tier  

### ISSUE THREAD

**Issue by GitPerseus** [noemica/cog-minder#23]
The "Transdimensional Reconstructors" part group page appears to be correctly marked as Spoiler-tier within the wiki.json file : 
https://github.com/noemica/cog-minder/blob/master/src/json/wiki.json#L7509

However, when using the wiki with spoiler settings set to None, the page still appears in search results and in the Non-empty Pages list, and going to the page itself does not trigger a "Page blocked by Spoilers setting" warning.
In other words, the page is considered to be at spoiler tier "None".

**Comment by noemica** [comment]
Ah yeah, I forgot that no part groups before now had any spoilers and so there was some code I needed to uncomment out. Fixed this in 38ec7b8, thanks for the report.
